examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/MainApplication.kt

```diff
@@ -22,7 +28,16 @@ class MainApplication : Application() {
                 .build()
         )
         Purchases.debugLogsEnabled = true
-        Purchases.configure(PurchasesConfiguration.Builder(this, "api_key").build())
+
+        val purchasesConfigurationBuilder =
+            when {
+                GOOGLE_API_KEY.isNotEmpty() -> PurchasesConfiguration.Builder(this, GOOGLE_API_KEY)
```

Comment:
Does this mean we can set two api keys at once? Like we could set both `GOOGLE_API_KEY` and `AMAZON_API_KEY` ?

Reply:
you could, and this would just prefer the google key. our recommendation in docs is to create product flavors, which i thought might be overkill for the purchase tester, but might be worth it?

Reply:
@taquitos gonna merge for now, but we have lots of other purchase tester tickets coming up to prep for amazon testing party, so we can definitely fix this soon if we want.